src/clients/api/queries/useGetPrimeEstimation/index.ts

```diff
@@ -0,0 +1,212 @@
+import BigNumber from 'bignumber.js';
```

Comment:
This file is wrapping multiple queries like `useGetPrimeDistributionForMarket` `useGetHypotheticalPrimeApys`. It is not making a unique query and contains a lot of extra logic. 

What about organizing this in under hooks?